client/src/components/main_section/main_section.coffee

```diff
@@ -12,7 +13,7 @@ SingleRepo = React.createFactory require '../repo_list/single_repo.coffee'
 
 MainSection = React.createClass
   render: ->
-    {repo, repoActions, login, pageLocation, pageLocationActions,loginActions} = @props
+    {repo, repoActions, login, pageLocation, pageLocationActions,loginActions, registerActions} = @props
```

Comment:
do your other components have access to dispatcher, so they can bind their own ActionCreators?